config/ksql-server.properties

```diff
@@ -66,6 +66,7 @@ ksql.logging.processing.stream.auto.create=true
 
 # The set of Kafka brokers to bootstrap Kafka cluster information from:
 bootstrap.servers=localhost:9092
+#ksql.source.table.materialization.enabled=false
```

Comment:
I think we should leave it out of the properties file. It could cause confusion, and we haven't been doing this generally for feature flags, right?.

Reply:
This shouldn't be here. I was testing and forgot to remove this part from the config. I'm probably going to mention this feature flag in the docs.